torch_tvm/operators.cpp

```diff
@@ -183,27 +217,60 @@ RegisterTVMOperator reg({
        conv_attrs->groups = relayToConstant<int>(inputs[8]);
        conv_attrs->data_layout = "NCHW";
        conv_attrs->kernel_layout = "OIHW";
+       conv_attrs->strides = relayToArray<tvm::relay::IndexExpr>(inputs[3]);
+       conv_attrs->padding = relayToArray<tvm::relay::IndexExpr>(inputs[4]);
+       conv_attrs->dilation = relayToArray<tvm::relay::IndexExpr>(inputs[5]);
 
        conv_attrs->kernel_size =
            tvm::NullValue<tvm::Array<tvm::relay::IndexExpr>>();
+       // For 1D conv need to expand dim. That means it has to be removed
+       // as well.
+       bool dim_added{false}; // For 1D conv need to expand dim.
```

Comment:
this is fine for now, but I wonder if we should be using a pass in PyTorch to make this more portable for other backends going forward? maybe add a TODO

Reply:
@bwasti, not sure what you mean. Do you mean make 1d conv look like 2D with jit pass and use such a pass if needed?